### PR TITLE
draft of Power updates

### PR DESCRIPTION
**Pull request by kalexand-rh**
These changes should go live on 9/18. In 3.11, Power won't be available until the second z stream release.

@jeffyoung, will you PTAL?

### CODE REVIEW

**Comment by kalexand-rh** [comment]
@jianlinliu, will you PTAL at this OpenShift on Power change?

File: install/index.adoc [hunk 1]
```diff
@@ -76,6 +82,23 @@ the provisioning and installation processes. For information, see
 xref:running_install.adoc#advanced-cloud-providers[Running Installation Playbooks].
 endif::[]
 
+[[install_power]]
+=== Limitations and Considerations for Installations on IBM POWER
+
+As of version 3.10.z, you can install {product-title} on IBM POWER servers.
```
Comment (kalexand-rh) [inline install/index.adoc]: I need to use the correct version number.

File: install/host_preparation.adoc [hunk 2]
```diff
@@ -17,14 +17,33 @@ ifdef::openshift-enterprise[]
...
 - https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/Installation_Guide/index.html[Red Hat Enterprise Linux 7 Installation Guide]
 - https://access.redhat.com/documentation/en/red-hat-enterprise-linux-atomic-host/version-7/installation-and-configuration-guide/[Red Hat Enterprise Linux Atomic Host 7 Installation and Configuration Guide]
 
+[[server-type]]
+== Server Type Requirements
+
+If you use IBM POWER servers for your nodes, you can use only IBM POWER servers.
+You cannot add nodes that run on IBM POWER servers to an existing cluster that
+uses x86_64 servers or deploy cluster nodes on a mix of IBM POWER and x86_64
+servers.
```
Comment (sferich888) [inline install/host_preparation.adoc]: This should also apply to the Masters or control infrastructure. Based on what we have been told in CEE

**Comment by kalexand-rh** [comment]
After we finish review, I need to open a new PR against just 3.10 instead of merging this one to master.

**Comment by jianlinliu** [comment]
>  will you PTAL at this OpenShift on Power change?

I only focus on x86 arch, some other QE would take care of powerpc individually. @brenton could you help involve some right person here?

**Comment by brenton** [comment]
Hi @jianlinliu , I don't know their github accounts but I'll email them a link to this PR and CC you.

**Comment by sferich888** [comment]
We need to look at lines like https://github.com/openshift/openshift-docs/pull/11915/files#diff-d93639ab27f9fd6f1e3c3366923a20a2R211 because the point directly to files; however with the addition of the power content these variables are not simply directory mappings anymore.

**Comment by kalexand-rh** [comment]
@sferich888, I don't see which lines you mean.

Because this change didn't make it into master before the 3.11 changes, I'm closing this PR and opening a new one against 3.10: #12272

**Pull request closed without merging**